src/socketio.quiver.js

```diff
@@ -617,6 +694,17 @@ QuiverSocialProvider.prototype.addFriend_ = function(servers, userId, nick, invi
  * @param {QuiverSocialProvider.server_} server
  * @private
  */
+QuiverSocialProvider.prototype.removeServer_ = function(server) {
+  var serverKey = QuiverSocialProvider.serverKey_(server);
+
+  delete this.configuration_.self.servers[serverKey];
```

Comment:
It seems like after removing a server from this.configuration_.self.servers, we should add a new random one.  Otherwise if removeServer_ gets called twice, won't this.configuration_.self.servers be empty?